src/lib/homestore_backend/replication_state_machine.hpp

```diff
@@ -33,6 +33,19 @@ struct repl_result_ctx : public ho_repl_ctx {
     folly::SemiFuture< T > result() { return promise_.getSemiFuture(); }
 };
 
+template < typename T >
+struct put_result_ctx : public repl_result_ctx< T > {
```

Comment:
Can we specialize `put_result_ctx` on `repl_result_ctx<BlobManager::Result< BlobInfo >>`? Is there a put that would be a different result?

Reply:
Now that I am using the `key` field for blob_id, I dont need the `put_result_ctx` anymore. I am removing this struct.